app/src/main/kotlin/com/metrolist/music/ui/screens/settings/AboutScreen.kt

```
@@ -330,7 +367,38 @@ fun AboutScreen(
+                                when {
+                                    remaining in 1..DEV_MODE_COUNTDOWN_START -> {
+                                        Toast.makeText(context, context.resources.getQuantityString(R.plurals.dev_mode_taps_remaining, remaining, remaining), Toast.LENGTH_SHORT).show()
+                                    }
+                                    remaining <= 0 -> {
+                                        tapCount = 0
+                                        lastTapTime = 0L
+                                        coroutineScope.launch {
+                                            try {
+                                                context.dataStore.edit { it[DeveloperModeKey] = true }
+                                                Toast.makeText(context, context.getString(R.string.dev_mode_enabled), Toast.LENGTH_LONG).show()
+                                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)